ui/library/src/components/BoundingBox.tsx

```diff
@@ -30,6 +32,7 @@ export const BoundingBox: React.FunctionComponent<Props> = ({
   const componentClassName = classNames(
     'reader__page-overlay__bounding-box',
     isHighlighted === true ? 'reader__page-overlay__bounding-box-highlighted' : '',
+    isPopoverVisible === true ? 'reader__page-overlay__bounding-box-selected' : '',
```

Comment:
Consider updating the `isPopoverVisible` prop name to something that matches the class name tied to it, for example `isSelected` or `isBoundingBoxSelected`.